linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_ca/ansible/shared.yml

```diff
@@ -1,22 +1,66 @@
-# platform = multi_platform_sle
+# platform = multi_platform_sle,multi_platform_ubuntu
```

Comment:
if having one shared ansible remediation, try to do the same with bash

Reply:
Replaced `sle.sh` and `ubuntu.sh` with `shared.sh`. Thank you!